order-service/src/app.ts

```diff
@@ -0,0 +1,33 @@
+import 'reflect-metadata';
+import DatabaseConfig from './config/database';
+import * as Express from 'express';
+import * as bodyParser from 'body-parser';
+
+class Core {

```

Comment:
We'll really be using this name? `OrderService` will conflict with `Order` service... Maybe `OrderApp`?

Reply:
Done